src/commands/createStaticWebApp/IStaticWebAppWizardContext.ts

```diff
@@ -13,6 +13,7 @@ export type CreateNewResource = { name?: string; html_url?: string };
 export interface IStaticWebAppWizardContext extends IResourceGroupWizardContext {
```

Comment:
Rather than adding "advancedCreation", just do this:

```suggestion
export interface IStaticWebAppWizardContext extends IResourceGroupWizardContext, ICreateChildImplContext {
```